src/js/ui/photoswipe-ui-default.js

```diff
@@ -707,9 +707,16 @@ var PhotoSwipeUI_Default =
 
 	ui.updateIndexIndicator = function() {
 		if(_options.counterEl) {
-			_indexIndicator.innerHTML = (pswp.getCurrentIndex()+1) + 
-										_options.indexIndicatorSep + 
-										_options.getNumItemsFn();
+			var current = pswp.getCurrentIndex(),
+				total = _options.getNumItemsFn(),
+				indicatorHTML;
+
+			if (_options.indexIndicatorFormatFn) {
+				indicatorHTML = _options.indexIndicatorFormatFn(current, total);
```

Comment:
a) `indexIndicatorSep` should be also included as parameter
b) `current` should be with `+1`